src/Logger/TextFileLogger.php

```diff
@@ -104,7 +104,7 @@ private function getMutatorFirstLine(int $index, MutantProcessInterface $mutantP
             $index + 1,
             $mutantProcess->getOriginalFilePath(),
             $mutantProcess->getOriginalStartingLine(),
-            $mutantProcess->getMutator()::getName()
+            $mutantProcess->getMutatorClass()::getName()
```

Comment:
```suggestion
            $mutantProcess->getMutatorClass()
```

Reply:
Hm it's not the same thing: `::getName()` will return `Plus` whereas `getMutatorClass()` will be the FQCN

Reply:
Yes, getMutatorClass needs to do that call apprently. And should it be renamed to getMutatorName?